imports/plugins/included/product-admin/client/components/TagSelector/Notifications.js

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import PropTypes from "prop-types";
+import InlineAlert from "@reactioncommerce/components/InlineAlert/v1";
+import { i18next } from "/client/api";
+import CloseIcon from "mdi-material-ui/Close";
+import { makeStyles } from "@material-ui/core";
+import classNames from "classnames";
```

Comment:
Use the `clsx` package instead. same thing, but faster.

Reply:
that package is not installed in Reaction as a direct dependency. It is in Catalyst, for now lets keep using classNames in Reaction until we decide to switch to `clsx`

Reply:
👍